src/effects/defs.h

```diff
@@ -28,6 +28,11 @@ enum class EffectChainInsertionType {
 
 const int kNumEffectsPerUnit = 4;
 
+// NOTE: Setting this to true will enable string manipulation and calls to
+// qDebug() in the audio engine thread. That may cause audio dropouts, so only
+// enable this when debugging the effects system.
+const bool kEffectDebugOutput = false;
```

Comment:
A constexpr might be more suitable to enforce evaluation (and optimization) at compile time